ansible/roles/eos/tasks/ceos_ensure_reachable.yml

```diff
@@ -0,0 +1,49 @@
+- block:
+  - name: wait for container's mgmt-ip reachable
+    wait_for:
+      port: 22
+      host: "{{ ansible_host }}"
+      search_regex: OpenSSH
+      delay: 0
+      timeout: 120
+    register: ip_result
+    ignore_errors: yes
+    delegate_to: "{{ VM_host[0] }}"
```

Comment:
Suggest to try the SSH connection from sonic-mgmt docker:
```
delegate_to: localhost
```

Reply:
done